disnake/ui/view.py

```diff
@@ -133,6 +131,10 @@ class View:
 
     This object must be inherited to create a UI within Discord.
 
+    Alternatively, components can be handled with :class:`disnake.ui.ActionRow`\\s and event
```

Comment:
are both `\\` needed?

Reply:
Yup, this seems to render fine on the docs and comes with the added benefit of not it crying over nonexistent escape sequences.